Mathlib/Data/Erased.lean

```diff
@@ -154,9 +154,8 @@ theorem map_def {α β} : ((· <$> ·) : (α → β) → Erased α → Erased β
 #align erased.map_def Erased.map_def
 
 --Porting note: Old proof `by refine' { .. } <;> intros <;> ext <;> simp`
-protected instance LawfulMonad : LawfulMonad Erased :=
-  { Erased.Monad with
-    id_map := by intros; ext; simp
+protected instance instLawfulMonad : LawfulMonad Erased :=
```

Comment:
Is it clear these are defeq?

Reply:
Yes -- `LawfulMonad` doesn't take any fields from `Erased.Monad` at all.

Reply:
I should have read below the fold.